docs/specs/openapi.yaml

```diff
@@ -0,0 +1,335 @@
+openapi: 3.1.0
+info:
+  title: Pebble API
+  version: v1
+paths:
+  /v1/changes/{id}/wait:
+    get:
+      summary: Wait for a change to complete
+      description: |
+        Wait for the change to be finished.
+        
+        If the wait operation succeeds, the returned Change.Err string will be non-empty if the change itself had an error.
```

Comment:
`Change.Err` is terminology from the Go client. This needs to say something like: "the result will have the `err` field set to an appropriate error message".